### PR TITLE
Fixes minor artifacts document issues

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Fixed a few minor document related issues for the artifact module.

## Description of the changes

- Use inline code block correctly.
- Fix indents of the Figure 4 block in the tutorial.

### CODE REVIEW

File: tutorial/20_recipes/012_artifact_tutorial.py [hunk 1]
```diff
@@ -429,8 +429,8 @@ def main():
 # .. list-table::
 #     :header-rows: 1
 #
-#    * - Fig 4. The chemical structure obtained by the above code.
-#    * - .. image:: https://github.com/optuna/optuna/assets/38826298/c6bd62fd-599a-424e-8c2c-ca88af85cc63
+#     * - Fig 4. The chemical structure obtained by the above code.
+#     * - .. image:: https://github.com/optuna/optuna/assets/38826298/c6bd62fd-599a-424e-8c2c-ca88af85cc63
```
Comment (nzw0301) [inline tutorial/20_recipes/012_artifact_tutorial.py:433]: Before: 
Screenshot 2024-07-22 at 14 10 42

Comment (nzw0301) [reply tutorial/20_recipes/012_artifact_tutorial.py:433]: After: 
Screenshot 2024-07-22 at 14 29 11

https://optuna--5592.org.readthedocs.build/en/5592/tutorial/20_recipes/012_artifact_tutorial.html

**Comment by nabenabe0928** [comment]
@nzw0301 
Thank you for the PR!

@not522 
Could you review this PR?

File: tutorial/20_recipes/012_artifact_tutorial.py [hunk 2]
```diff
@@ -86,17 +86,17 @@
...
-parameters were, when each trial started and ended, etc. This file is in the SQLite format, and it is not suitable for storing 
-large data. Writing large data entries may cause performance degradation. Note that SQLite is not suitable for distributed parallel 
-optimization. If you want to perform that, please use MySQL as we will explain later, or JournalStorage 
+Normally, Optuna's optimization history is persisted into some kind of a database via storage objects. Here, let's consider a
+method using SQLite, a lightweight RDB management system, as the backend. With SQLite, data is stored in a single file (e.g.,
+``./example.db``). The optimization history comprises what parameters were sampled in each trial, what the evaluation values for those
+parameters were, when each trial started and ended, etc. This file is in the SQLite format, and it is not suitable for storing
+large data. Writing large data entries may cause performance degradation. Note that SQLite is not suitable for distributed parallel
+optimization. If you want to perform that, please use MySQL as we will explain later, or ``JournalStorage``
```
Comment (not522) [inline tutorial/20_recipes/012_artifact_tutorial.py:94]: How about using cross-reference?

```suggestion
optimization. If you want to perform that, please use MySQL as we will explain later, or :class:`~optuna.storages.JournalStorage`
```

**Review by not522 — COMMENTED** [review]
Thank you for your PR. It's almost LGTM. Could you check my comment?

File: tutorial/20_recipes/012_artifact_tutorial.py (hunk 2 above)
Comment (nzw0301) [reply tutorial/20_recipes/012_artifact_tutorial.py]: Thanks! I've committed your suggestion and remove the following line done as https://github.com/optuna/optuna/pull/5592/commits/30fbb4ad941a774b515a49ee9a98708e35d5bb43.

**Review by not522 — APPROVED** [review]
LGTM!

**Review by nabenabe0928 — APPROVED** [review]
Thank you for the PR, LGTM!

**Pull request merged**